package.json

```diff
@@ -1,25 +1,18 @@
 {
   "name": "vss-web-extension-sdk",
-  "version": "1.108.0",
-  "description": "Visual Studio Services Client SDK. JavaScript library for Visual Studio Online extensions.",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "version": "1.109.0",
+  "description": "Visual Studio Services Client SDK. JavaScript library for Visual Studio Team Services extensions.",
   "repository": {
     "type": "git",
     "url": "https://github.com/Microsoft/vss-web-extension-sdk.git"
   },
+  "scripts": {
```

Comment:
Would it be worthwhile to just add a "prepublish" script that builds the package prior to publishing? This pattern is suggested by the NPM docs: https://docs.npmjs.com/misc/scripts